enterprise/frontend/src/embedding-sdk/components/private/Breadcrumb/SdkBreadcrumbs.unit.spec.tsx

```diff
@@ -0,0 +1,135 @@
+import { act, screen, waitFor, within } from "__support__/ui";
+
+import { setup } from "./test-setup";
+
+describe("SdkBreadcrumbs", () => {
```

Comment:
All this file and its complex setup look more like e2e test for me

Reply:
Yeah, agree that it smells more like an integration test. I wish we could name the file `SdkBreadcrumbs.integration.spec.tsx` as I think integration tests with Jest is also valid. I think Nemanja raised a good point in the FE guild weekly that we should try to use Jest tests more when possible, as it reduces flakiness and reduce the CI burden on our already hard-working Cypress CI.

Reply:
I asked in the [engineering-frontend](https://metaboat.slack.com/archives/C505ZNNH4/p1755251620728479) channel, just to see if more people are starting to use Jest more for integration tests recently, or if it's an idea that never took off.